### ISSUE TITLE
Suggest to replace the PDF files with perma-links to documentation.

### ISSUE THREAD

**Issue by ReinhardKeil** [Open-CMSIS-Pack/pack-examples#8]
Currently the DFP contains PDF files. We discussed to replace the PDF files with perma-links to documentation.

**Comment by jkrech** [comment]
This was done for the STM32U5xx_DFP only as a reference. 
Would these pack-examples be "deleted" once we have the separate repositories?

**Comment by jkrech** [comment]
@ZigaMahne can you please confirm that all the DFP/BSPs for STMicroelectronics use links to documentation instead of shipping pdf files?

**Comment by ZigaMahne** [comment]
@jkrech, 
Yes, I can confirm that all individual ST DFP/BSPs on [Open-CMSIS-Pack](https://github.com/Open-CMSIS-Pack) have updated to using links to the relevant documentation instead of including PDF files directly in the packages.
